system-tests/test/component_testing_spec.ts

```diff
@@ -116,15 +116,25 @@ describe(`Angular CLI versions`, () => {
 describe('svelte component testing', () => {
   systemTests.setup()
 
-  for (const bundler of ['webpack', 'vite']) {
-    systemTests.it(`svelte + ${bundler}`, {
-      project: `svelte-${bundler}-configured`,
-      testingType: 'component',
-      spec: '**/*.cy.js,!src/errors.cy.js',
-      browser: 'chrome',
-      expectedExitCode: 0,
-    })
-  }
+  // svelte-webpack-configured is currently difficult to test.
+  // This is currently tested as a binary-like system-test inside CI.
+  // We can unskip this test and remove the binary-like CircleCI test
+  // once https://github.com/sveltejs/svelte-loader/issues/243 is resolved.
```

Comment:
see the readme inside `svelte-webpack-configured` for why we need to do this. Main reason, `alias`ing svelte inside webpack doesn't work correctly, so the mount function being pulled in from `npm/svelte` is a different svelte instance than the one in `svelte-webpack-configured`. The work around for this is to use the binary (see circle workflow file) as this can't happen when the binary is used instead.

Reply:
Oh I get it more then. So this won't be an issue for our users right?

Reply:
correct this is just a problem with our system tests because of how we source the harnesses straight from the `./cli` package. installing the binary sidesteps this problem